app/services/nav-links.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
+
+
+export default Ember.Service.extend({
+
+    links: []
+
+});
```

Comment:
holds state about what nav links to show in the green bar. set by the route, and bound to the navbar.